internal/catalogmetadata/unmarshal.go

```diff
@@ -3,18 +3,16 @@ package catalogmetadata
 import (
 	"encoding/json"
 
-	catalogd "github.com/operator-framework/catalogd/api/core/v1alpha1"
+	"github.com/operator-framework/operator-registry/alpha/declcfg"
 )
 
-func Unmarshal[T Schemas](cm []catalogd.CatalogMetadata) ([]*T, error) {
-	contents := make([]*T, 0, len(cm))
-	for _, cm := range cm {
-		var content T
-		if err := json.Unmarshal(cm.Spec.Content, &content); err != nil {
+func Unmarshal[T Schemas](meta *declcfg.Meta) (*T, error) {
```

Comment:
It was convinient to convert `[]catalogd.CatalogMetadata` into `[]*Bundle` or `[]*Scheme` within another generic.

But given that now call it multipe times on its own & it does not operate on a slice - I think we should probably just get rid of it.

Reply:
Got rid of it in https://github.com/operator-framework/operator-controller/pull/411/commits/79cf2386f67dfe6578dcf757139dde69385c748d